test/it/uk/gov/hmrc/organisationsdetailsapi/cache/ShortLivedCacheSpec.scala

```diff
@@ -16,34 +16,32 @@
 
 package it.uk.gov.hmrc.organisationsdetailsapi.cache
 
-import java.util.UUID
-import org.scalatest.{BeforeAndAfterEach, TestSuite}
+import org.mongodb.scala.model.Filters
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AnyWordSpec
-import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import org.scalatest.wordspec.AsyncWordSpec
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.json.{JsString, Json, OFormat}
+import uk.gov.hmrc.mongo.play.json.Codecs.toBson
 import uk.gov.hmrc.mongo.test.MongoSupport
 import uk.gov.hmrc.organisationsdetailsapi.cache.ShortLivedCache
 import utils.TestSupport
 
+import java.util.UUID
 import scala.concurrent.ExecutionContext
 
 class ShortLivedCacheSpec
```

Comment:
We have created a cache repository so we do not need this test class? Do we have one to cover cache repository?